tests/MongoDB.Driver.Core.Tests/Core/Operations/MapReduceOperationTests.cs

```diff
@@ -405,7 +408,7 @@ public void CreateOutputOptions_should_return_expected_result()
                     BsonDocument.Parse("{ _id : 2, value : 4 }")
                 };
             }
-            results.Should().Equal(expectedResults);
+            results.Should().BeEquivalentTo(expectedResults);
```

Comment:
as it was discussed in the slack with @jyemin , I consider all these tests in the way when the order of returned documents is not specified.